_includes/footer.html

```diff
@@ -9,7 +9,7 @@ <h5>Rest.li</h5>
           <h5 class="black-text">Community</h5>
           <ul>
               <li>Github <a class="" href="https://github.com/linkedin/rest.li">Rest.li</a></li>
-              <li>Linkedin <a class="" href="http://www.linkedin.com/groups/Restli-4855943">Rest.li Group</a></li>
+              <li>Linkedin <a class="" href="https://www.linkedin.com/groups/4855943/">Rest.li Group</a></li>
```

Comment:
While you're at it, you should correct the capitalization here: `Linkedin -> LinkedIn`
You should also correct the capitalization in these other links: `Github -> GitHub`, `twitter -> Twitter`, `github issue tracking -> GitHub issue tracking`